### PR TITLE
Rebase smartWallet (#8488) to release-mainnet1B branch

### PR DESCRIPTION
**Pull request by Chris-Hibbert**
refs: #8488

## Description

Rebase #8488 on release-mainnet1B

### Security Considerations

None

### Scaling Considerations

None

### Documentation Considerations

N/A

### Testing Considerations

Verify that the tests all pass. 

### Upgrade Considerations

It's all about preparing for upgrade.

### CODE REVIEW

**Comment by Chris-Hibbert** [comment]
Many tests are failing because the fix for fakeVirtualSupport to handle watchedPromises doesn't work here. The `vrm.allocateNextID()` call doesn't exit here.

File: packages/boot/test/bootstrapTests/liquidation.ts
Comment (mhofman) [inline packages/boot/test/bootstrapTests/liquidation.ts:1]: I believe the `boot` package doesn't exist in the release branch

File: packages/inter-protocol/src/price/roundsManager.js
Comment (mhofman) [inline packages/inter-protocol/src/price/roundsManager.js]: Seems like an unnecessary change for the release branch

File: packages/swingset-liveslots/src/watchedPromises.js
Comment (mhofman) [inline packages/swingset-liveslots/src/watchedPromises.js:1]: Any chance we can avoid these non-runtime changes in the release branch ?

File: packages/swingset-liveslots/src/types.js
Comment (mhofman) [inline packages/swingset-liveslots/src/types.js:1]: Any chance we can avoid these non-runtime changes in the release branch ?

File: packages/inter-protocol/src/price/roundsManager.js
Comment (Chris-Hibbert) [reply packages/inter-protocol/src/price/roundsManager.js]: dropped.

File: packages/boot/test/bootstrapTests/liquidation.ts
Comment (Chris-Hibbert) [reply packages/boot/test/bootstrapTests/liquidation.ts:1]: thx.  dropped.

File: packages/swingset-liveslots/src/watchedPromises.js
Comment (Chris-Hibbert) [reply packages/swingset-liveslots/src/watchedPromises.js:1]: removed.

File: packages/inter-protocol/src/price/roundsManager.js
Comment (Chris-Hibbert) [reply packages/inter-protocol/src/price/roundsManager.js]: removed.

File: packages/swingset-liveslots/src/types.js
Comment (Chris-Hibbert) [reply packages/swingset-liveslots/src/types.js:1]: removed.

File: packages/smart-wallet/package.json
Comment (mhofman) [inline packages/smart-wallet/package.json]: On the release branch, this is the version of `deploy-script-support`.

```suggestion
    "@agoric/deploy-script-support": "^0.10.4-u13.0",
```

Comment (Chris-Hibbert) [reply packages/smart-wallet/package.json]: done

**Review by turadg — COMMENTED** [review]
I reviewed https://github.com/Agoric/agoric-sdk/pull/8488#pullrequestreview-1784658822 . 

I think that should land first. Once it lands, the commit should be rebased here. I will be on PTO at that point.

File: packages/inter-protocol/src/price/roundsManager.js
Comment (mhofman) [inline packages/inter-protocol/src/price/roundsManager.js]: Seems like an unnecessary change

File: packages/inter-protocol/src/vaultFactory/vaultManager.js
Comment (mhofman) [inline packages/inter-protocol/src/vaultFactory/vaultManager.js]: Is the vault factory getting upgraded in this ? If not, this change should not be part of the dev branch

**Review by mhofman — DISMISSED** [review]
Looks like there are some spurious changes unrelated to wallet factory in this PR

File: packages/inter-protocol/src/price/roundsManager.js
Comment (Chris-Hibbert) [reply packages/inter-protocol/src/price/roundsManager.js]: removed

File: packages/inter-protocol/src/vaultFactory/vaultManager.js
Comment (Chris-Hibbert) [reply packages/inter-protocol/src/vaultFactory/vaultManager.js]: dropped

**Review by mhofman — COMMENTED** [review]
I have not reviewed the logic, nor if the rebase is faithful, but the scope of modifications is acceptable for release.

**Review by turadg — APPROVED** [review]
I assume the commits are the same as master. The reconciliation commit looks okay. There's a merge conflict to solve but I trust it will be safe.

**Comment by Chris-Hibbert** [comment]
Given recent discussions, I will wait for a new release branch before proceeding.

**Comment by Chris-Hibbert** [comment]
superseded by #8773.

**Pull request closed without merging**
